### PR TITLE
sentiment score for tag/latlong

### PR DESCRIPTION
**Pull request by SrishtiR**
Code to return sentiment score, init files in constants and helpers folder

### CODE REVIEW

**Comment by skinnerr** [comment]
Do we need to have the binary `.pyc` files in the git repository? I would prefer we exclude them since python re-generates them from the `.py` files each time they are needed.

File: rest-twitter/service/TweetRetriever_Test.py [hunk 1]
```diff
@@ -2,10 +2,31 @@
...
 
+
 from TweetRetriever import TweetRetriever
+from vaderSent import SentimentAnalyzer
+from config.location_service_config import CACHE
+from Location_Service import *
+import time
+import os
+from computeSent import returnSent
+# tr = TweetRetriever()
```
Comment (AbigailFernandes) [inline rest-twitter/service/TweetRetriever_Test.py]: Do not commit commented code

File: rest-twitter/service/TweetRetriever_Test.py [hunk 2]
```diff
@@ -2,10 +2,31 @@
...
 
-tweets = tr.get_tweets('cold','39.7392358','-104.990251')
+# for each in tweets:
+#     score = vs.sentimentAnalyzerScores(each)
+#     print(each, score)
+#     print("#####")
+# avgSent = vs.computeSentiment(tweets)
+# print(avgSent)
 
-print(tweets)
```
Comment (AbigailFernandes) [inline rest-twitter/service/TweetRetriever_Test.py]: Is this print statement required? Print statements that are kept for debugging purposes should be removed while commiting

File: rest-twitter/service/TweetRetriever_Test.py [hunk 3]
```diff
@@ -2,10 +2,31 @@
...
+avgsent = returnSent('US', 'Denver', 'hello')
+print(avgsent)
```
Comment (AbigailFernandes) [inline rest-twitter/service/TweetRetriever_Test.py]: Do not print unless its an error message which we will need for logging purposes

File: rest-twitter/service/computeSent.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+from TweetRetriever import TweetRetriever
+from vaderSent import SentimentAnalyzer
+from config.location_service_config import CACHE
+from Location_Service import *
+import time
+import os
+import json
+import requests
+
+def returnSent(country, city, hashtag):
```
Comment (AbigailFernandes) [inline rest-twitter/service/computeSent.py]: Name the function 'compute_sentiment

File: rest-twitter/service/vaderSent.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+                analyser = SentimentIntensityAnalyzer()
+                score = analyser.polarity_scores(sentence)
+                return score['compound']
+
+        def computeSentiment(self,tweets):
+                compoundSum = 0
+                nTweets = 0
+                if len(tweets) == 0:
+                        return 0
+                for each in tweets:
```
Comment (AbigailFernandes) [inline rest-twitter/service/vaderSent.py]: should be `for tweet in tweets`
Use descriptive names

File: rest-twitter/service/vaderSent.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+class SentimentAnalyzer:
+        def sentimentAnalyzerScores(self,sentence):
+                analyser = SentimentIntensityAnalyzer()
+                score = analyser.polarity_scores(sentence)
+                return score['compound']
+
+        def computeSentiment(self,tweets):
+                compoundSum = 0
+                nTweets = 0
```
Comment (AbigailFernandes) [inline rest-twitter/service/vaderSent.py]: Could use `num_tweets = len(tweets)`.
Don't need to use a counter within the for loop

File: rest-twitter/service/vaderSent.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+                if len(tweets) == 0:
+                        return 0
```
Comment (AbigailFernandes) [inline rest-twitter/service/vaderSent.py]: Does score = 0 indicate some error or does 0 mean a sentiment

File: rest-twitter/service/computeSent.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
+from TweetRetriever import TweetRetriever
```
Comment (AbigailFernandes) [inline rest-twitter/service/computeSent.py]: Rename this file to compute_sentiment

File: rest-twitter/service/computeSent.py [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+import json
+import requests
+
+def returnSent(country, city, hashtag):
+    tr = TweetRetriever()
+    vs = SentimentAnalyzer()
+    path = os.path.join(os.path.curdir, CACHE['location'])
+    ls = LocationService(path, CACHE['dump_interval'])
+    coords = ls.get_coordinates_for_city({'city': city, 'country': country})
+    print(coords)
```
Comment (shreshtha-pankaj) [inline rest-twitter/service/computeSent.py]: Do we need to log the coords?

File: rest-twitter/service/computeSent.py [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+    tweets = tr.get_tweets(hashtag,coords[LAT],coords[LON])
+    print(tweets)
```
Comment (shreshtha-pankaj) [inline rest-twitter/service/computeSent.py]: Do we need to log the tweets?

File: rest-twitter/service/TweetRetriever_Test.py (hunk 2 above)
Comment (SrishtiR) [reply rest-twitter/service/TweetRetriever_Test.py]: removed the test file

File: rest-twitter/service/TweetRetriever_Test.py (hunk 3 above)
Comment (SrishtiR) [reply rest-twitter/service/TweetRetriever_Test.py]: removed the test file

File: rest-twitter/service/computeSent.py (hunk 4 above)
Comment (SrishtiR) [reply rest-twitter/service/computeSent.py]: changed to compute_sentiment

File: rest-twitter/service/vaderSent.py (hunk 6 above)
Comment (SrishtiR) [reply rest-twitter/service/vaderSent.py]: removed the counter

File: rest-twitter/service/vaderSent.py (hunk 7 above)
Comment (SrishtiR) [reply rest-twitter/service/vaderSent.py]: changed to "No tweet found"

File: rest-twitter/service/computeSent.py (hunk 8 above)
Comment (SrishtiR) [reply rest-twitter/service/computeSent.py]: renamed

File: rest-twitter/service/computeSent.py (hunk 9 above)
Comment (SrishtiR) [reply rest-twitter/service/computeSent.py]: No, removed the print statements

File: rest-twitter/service/computeSent.py (hunk 10 above)
Comment (SrishtiR) [reply rest-twitter/service/computeSent.py]: Removed the print statement

**Review by AbigailFernandes — APPROVED** [review]

**Pull request merged**